$OpenBSD: patch-src_ObservableQueue_h,v 1.1 2012/05/09 12:51:01 dcoppa Exp $

Fix compilation with gcc 4.7
(upstream git commit 4a75ae360d51d8370445da01461d7fbdf2c41641)

--- src/ObservableQueue.h.orig	Wed May  9 13:13:53 2012
+++ src/ObservableQueue.h	Wed May  9 13:14:39 2012
@@ -331,14 +331,14 @@ CObservableQueue<ValueType>::~CObservableQueue()
 template <typename ValueType>
 void CObservableQueue<ValueType>::ObserverAdded( ObserverType* o )
 {
-	NotifyObservers( EventType( EventType::STARTING ), o );
+	this->NotifyObservers( EventType( EventType::STARTING ), o );
 }
 
 
 template <typename ValueType>
 void CObservableQueue<ValueType>::ObserverRemoved( ObserverType* o )
 {
-	NotifyObservers( EventType( EventType::STOPPING ), o );
+	this->NotifyObservers( EventType( EventType::STOPPING ), o );
 }
 
 
